src/sql/workbench/contrib/editData/browser/editDataResultsEditor.ts

```diff
@@ -50,13 +50,17 @@ export class EditDataResultsEditor extends BaseEditor {
 	}
 
 	public get input(): EditDataResultsInput {
-		return this._input;
+		return this._input as EditDataResultsInput;
```

Comment:
probably don't need to add a cast here since `_input` is declared as `EditDataResultsInput` already